### PR TITLE
Add jquery.basictable@1.0.6 w/ git auto-update

### LINKED ISSUE #9575: [Request] Add basictable.min.js
**Library name:** basictable
**Git repository** https://github.com/jerrylow/basictable/blob/master/jquery.basictable.js
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #9575, cc @jerrylow, using web-minify-helper tool for minifing css files

Pull request for issue: #9575
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jerrylow/basictable
 * Official website (optional, not the repository): https://www.jerrylow.com/basictable/
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 4
   - Count of stars: 45
   - Count of forks: 19
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery.basictable/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery.basictable",
+  "version": "1.0.6",
+  "filename": "jquery.basictable.min.js",
+  "homepage": "https://www.jerrylow.com/basictable/demo/",
```
Comment (extend1994) [inline ajax/libs/jquery.basictable/package.json]: Please use `https://www.jerrylow.com/basictable/`. 
With `demo`, it only shows demo. 
Without `demo`, users can see intro, setup, link to demo and so on.

File: ajax/libs/jquery.basictable/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "git://github.com/jerrylow/basictable.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jerrylow/basictable.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*basictable.*+(js|css)"
```
Comment (extend1994) [inline ajax/libs/jquery.basictable/package.json]: .map files could appear in the future. Please use `*basictable.*+(js|css|map)` instead.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@sufuf3 Thanks a lot

**Comment by PeterDaveHello** [comment]
Please do not generate map files in this case in the future as there is no map file for js files, thanks.

**Pull request merged**